app/build.gradle

```diff
@@ -105,60 +104,66 @@ play {
 }
 
 detekt {
-    input = "./src/main/java"
-    config = "$rootDir/team-props/static-analysis/detekt-config.yml"
-    filters = ".*test.*, .*/resources/.*"
-    output = true
-    report = "$project.projectDir/build/reports"
+    version = "1.0.0.M13"
+
+    profile("main") {
+        input = "$projectDir/src/main/java"
+        config = "$rootDir/team-props/static-analysis/detekt-config.yml"
+        filters = ".*test.*,.*/resources/.*,.*/tmp/.*"
+        output = "$project.projectDir/build/reports/detekt.xml"
```

Comment:
Cause I did not really change this line, but I did the other one, and in this one `project.` is unnecessary :)